### PR TITLE
928 fix isinperson fileds

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: utilities/sourceEvents.py [hunk 1]
```diff
@@ -145,7 +145,10 @@ def parse(content, gmaps):
         if 'virtualEventURL' in pe:
             entry['virtualEventUrl'] = pe['virtualEventURL']
         if 'inPersonEvent' in  pe:
-            entry['isInPerson'] = pe['inPersonEvent']
+            if pe['inPersonEvent'] == "true":
```
Comment (sandeep-ps) [inline utilities/sourceEvents.py:148]: I think we might have to add the else case here for `isVirtualEvent` field in line 143. Currently, it looks like it ignores if the value from WebTools is false.

Comment (bingzhang) [reply utilities/sourceEvents.py:148]: what is the value in else?

Comment (sandeep-ps) [reply utilities/sourceEvents.py:148]: I meant that this if-else approach that you are using here might be needed for isVirtualEvent field too. Currently, for isVirtualEvent, we are setting true if the incoming value is true. I didn't see if we were setting false by default for the alternate case. Does this explain better?

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
I'm going to merge this PR. 

@bingzhang, please deploy the latest changes to DEV. Thanks.

**Comment by sandeep-ps** [comment]
Bing, for future PRs, please make sure to create the branch from GitHub. The current PR has 100+ commits, probably from the local master branch. I will merge this anyway. Thanks.

**Comment by bingzhang** [comment]
[Calendar events with the new fields](https://github.com/rokwire/events-manager/issues/928)

**Pull request merged**